src/Configuration/Config.php

```diff
@@ -22,15 +26,17 @@ public function getDatabases(): array
         return $this->getArrayValue(['parameters', 'migrateDatabases']);
     }
 
-    public function getSynchronizeRun(): bool
+    public function isSynchronizeRun(): bool
     {
-        return is_array($this->getValue(['parameters', 'synchronize'], false));
+        /** @var bool $value */
+        $value = $this->getValue(['parameters', 'synchronize'], false);
+        return $value;
```

Comment:
great!  I never liked `return is_array($this->getValue(['parameters', 'synchronize'], false));` it was confusing for my old man brain 😄